src/main/java/pages/AbstractPage.java

```diff
@@ -36,5 +36,6 @@ protected boolean waitForElementIsPresent(WebElement element) {
 
         return element.isEnabled();
     }
+
```

Comment:
empty line

Reply:
Deleted, thank you for pointing.
Please see in the following commit.